index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav class="navbar navbar-expand-sm bg-light fixed-top" id="navbar">
+  
+        <ul class="navbar-nav ml-auto text-white" style="font-size: 1.5rem;">
+            <li class="nav-item">
+                <a class="nav-link text-white" href="#welcome-section">
+                  About
+                </a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link text-white" href="#projects">
+                  Work
+                </a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link text-white" href="#contact">
+                  Contact
+                </a>
+            </li>
+        </ul>
+    </nav>
+    
+    <div class="w-100 h-100">
+
+      <section id='welcome-section'>
+        <div class="welcome-title text-center h-100 align-middle  vertical-center">
+          <div class="container">
+            <h1 class="heading">Hey I am Omar & Saadat</h1>
+            <p class="sub-heading">Aspiring Web Developers</p>
+          </div>
+        </div>
+      </section>
+
+      <section id='projects'>
+        <h1 class="text-white text-center" style="margin-bottom: 6rem;">
+          <span class="border-bottom">These are some of my projects</span>
+        </h1>
```

Comment:
I think h1 should only be used once throughout the document. It helps in SEO of the website. Use h2-h5 instead.
```suggestion
        <h2 class="text-white text-center" style="margin-bottom: 6rem;">
          <span class="border-bottom">These are some of my projects</span>
        </h2>
```